include/seqan3/range/decorator/gap_decorator_anchor_set.hpp

```diff
@@ -369,7 +369,8 @@ class gap_decorator_anchor_set
     ~gap_decorator_anchor_set() = default;
 
     //!\brief Construct with the ungapped range type.
-    constexpr gap_decorator_anchor_set(inner_type const & range) : ungapped_view{view::all(range)} {}
+    gap_decorator_anchor_set(inner_type range) : ungapped_view{view::all(std::forward<inner_type>(range))}
+    {}
     // !\}
 
```

Comment:
Is it safe to use one interface here? This function also shouldn't accept container temporaries...

Reply:
this was your code ^^

Reply:
If I try to give him a temporary it says `type deduction failed`.. not a very pretty error

Reply:
It cannot accept container temporaries otherwise the inner_type would be ill-formed due to the ViewableRange constraint. 🤔

Reply:
yes and that is the behaviour we want